server/proofs.go

```diff
@@ -64,78 +38,59 @@ func (v *VersionedSignedBuilderBidWithProofs) MarshalJSON() ([]byte, error) {
 			Signature: v.Deneb.Signature,
 			Proofs:    v.Proofs,
 		})
+	default:
+		return nil, fmt.Errorf("unknown or unsupported data version %d", v.Version)
 	}
-
-	return nil, fmt.Errorf("unknown data version %d", v.Version)
 }
 
+// this is necessary, because the mev-boost-relay deserialization doesn't expect a "Version" and "Data" wrapper object
```

Comment:
Love the log

Reply:
Oh wait, that should be removed actually ahah